admin/sql/CreateReplicationTriggers.sql

```diff
@@ -99,6 +99,10 @@ CREATE TRIGGER "reptg_artist_credit"
 AFTER INSERT OR DELETE OR UPDATE ON "artist_credit"
 FOR EACH ROW EXECUTE PROCEDURE "recordchange" ();
 
+CREATE TRIGGER "reptg_artist_credit_gid_redirect"
+AFTER INSERT OR DELETE OR UPDATE ON "artist_credit_gid_redirect"
+FOR EACH ROW EXECUTE PROCEDURE "recordchange" ('verbose');
+
```

Comment:
We'll also need a reptg2 trigger for dbmirror2 if/once we merge that (we should probably review the dbmirror2 triggers once we've done merging all schema change code).